src/components/InlineNotification/InlineNotification.module.css

```diff
@@ -0,0 +1,88 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # INLINE NOTIFICATION
+\*------------------------------------*/
+
+/**
+ * A messaging element that is used inline.
+ */
+.inline-notification {
+  display: inline-flex;
+  padding: var(--eds-size-half);
+  border: var(--eds-theme-border-width) solid var(--border-dark-color);
+  border-radius: var(--eds-theme-border-radius);
+}
+
+/**
+ * Icon that sits left of the notification text.
+ * 1) Retain icon original size and wrap text instead.
+ */
+.inline-notification__icon {
+  flex-shrink: 0; /* 1 */
+}
+
+/**
+ * Text that sits right of the icon.
+ */
+.inline-notification__text {
+  margin: var(--eds-size-half);
+  @mixin eds-theme-typography-label-md-subtle;
+}
+
+/**
+ * Color variants
+ */
+.inline-notification--brand {
+  @mixin messagingBrand;
+}
+.inline-notification--success {
+  @mixin messagingSuccess;
+}
+.inline-notification--warning {
+  @mixin messagingWarning;
+}
+.inline-notification--yield {
+  @mixin messagingRevise;
+}
```

Comment:
I find the "yield" vs "revise" bit here a little confusing. I feel like if the variant is called "yield", that's what the mixin should use even though the color token it's using is called "revise".

Reply:
Sorry to get all annoying about these naming details 😓

Reply:
It's so confusing will rename to `messagingYield`